client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.tsx

```diff
@@ -142,7 +147,7 @@ export function useProcessingLoadingMessages( flow?: string | null ): LoadingMes
 			{ title: __( "Let's head to the checkout" ), duration: 5000 },
 		];
 		return videoPressLoadingMessages;
-	} else if ( VIDEOPRESS_TV_FLOW === flow ) {
+	} else if ( VIDEOPRESS_TV_FLOW === flow || VIDEOPRESS_TV_PURCHASE_FLOW === flow ) {
```

Comment:
I think this should have its own messages probably?

`{ title: __( "Let's head to the checkout" ), duration: 5000 },` from the original flow feels most relevant if we're going to be doing some kind of API request to get checkout details before sending the customer to the checkout page?

Reply:
Added separate mesage block!